### PR TITLE
Fix for issue #166 and more for Laravel 5.2

### PR DESCRIPTION
**Pull request by Rainesdave**
this branch fix issue #166, #163  and multiple session creation for single user. I basically use laravel 5.2 but it should still work with 4|5 versions

### CODE REVIEW

**Comment by antonioribeiro** [comment]
So it would not be compatible with Laravel 4 anymore?

**Comment by Rainesdave** [comment]
it should still work, not tested though. I never use Laravel 4

**Comment by aftercares** [comment]
When I updated my project of Laravel 5.2 as mentioned above, I get the following error:

ErrorException in ClassLoader.php line 412:
include(/Applications/MAMP/htdocs/aftercares/vendor/pragmarx/tracker/src/Vendor/Laravel/Facade.php): failed to open stream: No such file or directory

**Comment by tarzanking** [comment]
@Rainesdave I have try use your method which is add \PragmaRX\Tracker\Vendor\Laravel\TrackerMiddleware::class, into kernel
but the user_id still getting null

please help ... and thanks

**Comment by gyleodhis** [comment]
Hello aftercares can you help me with how you integrated the tracker to laravel 5.2. I really do not know how to go around this issue. I have downloaded the tracker manually but adding it to my laravel project is a nightmare; some help please.

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "pragmarx/tracker",
+    "name": "rainestech/tracker",
```
Comment (antonioribeiro) [inline composer.json]: Doesn't look like right to me.

File: composer.json [hunk 2]
```diff
@@ -9,8 +9,8 @@
 
     "authors": [
         {
-            "name": "Antonio Carlos Ribeiro",
-            "email": "acr@antoniocarlosribeiro.com",
+            "name": "Raines Dave",
```
Comment (antonioribeiro) [inline composer.json]: You're not really the author, right?

**Pull request closed without merging**